### DISCUSSION TITLE
Sequent Microsystems MEGA-BAS Out of Stock

### DISCUSSION THREAD

**Discussion by mhmatttu** [tagyoureit/nodejs-poolController#448] [General]
I'm excited to get REM Chem going.  I don't believe I need the MEGA-BAS to get up and running but would prefer to build out the system with it for its RS-485, power supply, inputs, etc. and for future system expansion.  It has been out of stock for a while.  Any idea if these cards re-stock often or may be out of stock for a while?  Thanks!

https://sequentmicrosystems.com/collections/all-io-cards/products/raspberry-pi-building-automation?variant=37993168699587

**Comment by rstrouse** [comment]
Shoot them an e-mail.  They can give you some idea of their parts wait.  They   The BAS is really a good card.
